src/sentry/static/sentry/app/components/events/meta/metaProxy.tsx

```diff
@@ -12,6 +14,8 @@ function isAnnotated(meta) {
 }
 
 export class MetaProxy {
+  private local: any;
```

Comment:
I believe that local could be of type object

Reply:
I've checked the values that the constructor receives and it can be of type object or undefined 

maybe you could add a type like: `type Local = Record<string, any> | undefined;`